### PR TITLE
SS 416 add an about page to 0.7

### PR DESCRIPTION
**Pull request by sandstromviktor**
## Status

- [X] Ready
- [ ] Draft
- [ ] Hold

## Description
SS-416

I've added an `about.html` to `portal/templates/portal`. The `html` file comes from the current 0.5 beta version. I've changed the layout to better match 0.7 using some bootstrap stuff. Should be revised later i think. 

I've added the about page to the Django infrastructure such that the links work properly. See `views.py` and `urls.py`. 
When setting up local dev using docker compose, this works just fine. 

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [x] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [X] This pull request is against **develop** branch (not applicable for hotfixes)
- [X] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [ ] I have included tests to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [X] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request

## Further comments

I've changed the version of PyYaml from 6.0 to 6.0.1 because it threw errors regarding `cython_sources` during the build process. Seems to be a [known error](https://github.com/yaml/pyyaml/issues/601)

### CODE REVIEW

File: portal/templates/portal/about.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            </div>
+            </div>
+        </div>
+        </div>
+
+
+{% endblock %}
+
+
+
```
Comment (churnikov) [inline portal/templates/portal/about.html:60]:
```suggestion
```

Comment (sandstromviktor) [reply portal/templates/portal/about.html:60]: Ah yes. Will fix

File: portal/urls.py [hunk 2]
```diff
@@ -7,5 +7,6 @@
 urlpatterns = [
     path("portal/index", views.index, name="index"),
     path("portal/home", views.HomeView.as_view(), name="home"),
-    path("", views.HomeView.as_view(), name="about"),
+    path("portal/about", views.about, name="about"),
+    path("", views.HomeView.as_view(), name="about-page"),
```
[the remark this answers is not in the document]
Comment (sandstromviktor) [reply portal/urls.py:11]: I think so!

[the remark this answers is not in the document]
Comment (churnikov) [reply portal/urls.py:11]: It just that I don't really like the naming of `"home"` and `"about-page"`
I would personally like them to be under the same name `"home"`

But may be we could name them `"home"` and `"home-page"`

[Or use regex url pattern (but I'm not insisting on this)](https://stackoverflow.com/questions/47947673/is-it-better-to-use-path-or-url-in-urls-py-for-django-2-0)

**Review by churnikov — APPROVED** [review]

**Review by alfredeen — APPROVED** [review]
Good work. I personally liked the font and styles of href links better in the old version but that will be an oversight of the general site css I suppose.

**Pull request merged**